src/core.ts

```diff
@@ -16,6 +16,9 @@ export function createMagicMoveMachine(
   let current = EMPTY
 
   function commit(code: string): { current: KeyedTokensInfo, previous: KeyedTokensInfo } {
+    if (code === current.code)
```

Comment:
I have to revert it - it make theme only changes not applicable. Better to fix react instead

Reply:
Got it. I will need to make it work with React StrictMode